pkg/hub/managedcluster/controller.go

```diff
@@ -160,37 +160,37 @@ func (c *spokeClusterController) sync(ctx context.Context, syncCtx factory.SyncC
 		acceptedCondition.Message = applyErrors.Error()
 	}
 
-	_, updated, updatedErr := helpers.UpdateSpokeClusterStatus(
+	_, updated, updatedErr := helpers.UpdateManagedClusterStatus(
 		ctx,
 		c.clusterClient,
-		spokeClusterName,
-		helpers.UpdateSpokeClusterConditionFn(acceptedCondition),
+		managedClusterName,
+		helpers.UpdateManagedClusterConditionFn(acceptedCondition),
 	)
 	if updatedErr != nil {
 		errs = append(errs, updatedErr)
 	}
 	if updated {
-		c.eventRecorder.Eventf("SpokeClusterAccepted", "spoke cluster %s is accepted by hub cluster admin", spokeClusterName)
+		c.eventRecorder.Eventf("ManagedClusterAccepted", "spoke cluster %s is accepted by hub cluster admin", managedClusterName)
 	}
 	return operatorhelpers.NewMultiLineAggregate(errs)
 }
 
-func (c *spokeClusterController) removeSpokeClusterResources(ctx context.Context, spokeClusterName string) error {
-	err := c.kubeClient.CoreV1().Namespaces().Delete(ctx, spokeClusterName, metav1.DeleteOptions{})
+func (c *managedClusterController) removeManagedClusterResources(ctx context.Context, managedClusterName string) error {
+	err := c.kubeClient.CoreV1().Namespaces().Delete(ctx, managedClusterName, metav1.DeleteOptions{})
 	if err != nil && !errors.IsNotFound(err) {
 		return err
 	}
-	c.eventRecorder.Eventf("SpokeClusterNamespaceDeleted", "namespace %s is deleted", spokeClusterName)
+	c.eventRecorder.Eventf("ManagedClusterNamespaceDeleted", "namespace %s is deleted", managedClusterName)
 
-	clusterRoleName := fmt.Sprintf("%s:%s", clusterRolePrefix, spokeClusterName)
+	clusterRoleName := fmt.Sprintf("%s:%s", clusterRolePrefix, managedClusterName)
 	err = c.kubeClient.RbacV1().ClusterRoles().Delete(ctx, clusterRoleName, metav1.DeleteOptions{})
 	if err != nil && !errors.IsNotFound(err) {
 		return err
 	}
 	c.eventRecorder.Eventf("SpokeClusterClusterRoleDeleted", "clusterrole %s is deleted", clusterRoleName)
 
 	//TODO search all clusterroles and roles for this group and remove the entry or delete the clusterrolebinding if it's the only subject.
-	clusterRoleBindingName := fmt.Sprintf("%s:%s", clusterRolePrefix, spokeClusterName)
+	clusterRoleBindingName := fmt.Sprintf("%s:%s", clusterRolePrefix, managedClusterName)
 	err = c.kubeClient.RbacV1().ClusterRoleBindings().Delete(ctx, clusterRoleBindingName, metav1.DeleteOptions{})
 	if err != nil && !errors.IsNotFound(err) {
 		return err
```

Comment:
Two lines below (github won't let me add a comment there), there's an event type that still says `SpokeCluster`